### PR TITLE
Fix postgres add reference

### PR DESCRIPTION
**Pull request by parsonsmatt**
Before submitting your PR, check that you've:

- [x] Documented new APIs with [Haddock markup](https://www.haskell.org/haddock/doc/html/index.html)
- [x] Added [`@since` declarations](http://haskell-haddock.readthedocs.io/en/latest/markup.html#since) to the Haddock
- [x] Ran `stylish-haskell` on any changed files.
- [x] Adhered to the code style (see the `.editorconfig` file for details)

After submitting your PR:

- [x] Update the Changelog.md file with a link to your PR
- [x] Bumped the version number if there isn't an `(unreleased)` on the Changelog
- [ ] Check that CI passes (or if it fails, for reasons unrelated to your change, like CI timeouts)

### CODE REVIEW

**Comment by parsonsmatt** [comment]
Closed #1279

File: persistent-postgresql/Database/Persist/Postgresql.hs [hunk 1]
```diff
@@ -1268,7 +1268,7 @@ findAlters defs edef col@(Column name isNull sqltype def _gen _defConstraintName
                             | Just _oldName /= fmap fieldDB (getEntityIdField edef)
                             ->
                             [AddReference
-                                (getEntityDBName edef)
+                                (crTableName colRef)
```
Comment (parsonsmatt) [inline persistent-postgresql/Database/Persist/Postgresql.hs:1271]: This is the actual fix lol

**Comment by parsonsmatt** [comment]
@codedmart want to give this a shot?

**Comment by codedmart** [comment]
Ok I will try this out tomorrow morning. Thanks!

**Comment by codedmart** [comment]
@parsonsmatt Works perfect. Thanks, much appreciated!

**Pull request merged**